Create ui-c8y-1020-6-0-added-show-child-alarms-toggle-for-the-alarm-list-widget-view.md (#2262)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-6-0-added-show-child-alarms-toggle-for-the-alarm-list-widget-view.md

---
date: ""

> title: Alarm list widget configuration now supports showing and hiding child alarms

Reviewer: ```suggestion
title: Alarm list widget configuration now supports showing or hiding child alarms
```

> title: Added showChildAlarms toggle for the alarm list widget view to improve user experience

Author: The Claude AI suggested to use the following title:
```suggestion
title: Alarm list widget view now supports showing and hiding child alarms
```

> title: Added showChildAlarms toggle for the alarm list widget view to improve user experience

Reviewer 2: ```suggestion
title: Alarm list widget configuration now supports showing and hiding child alarms
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-55763
version: 1020.6.0
---

> Added showChildAlarms toggle for the alarm list widget view to improve user experience

Author: The Claude AI suggested to use the following body:
```suggestion
The alarm list widget view is used to display alarms in dashboards and other places in the Cumulocity user interface. Previously, child alarms were always shown which could lead to information overload for users monitoring many alarms. With this change, a new toggle has been added to the widget configuration which allows showing or hiding the child alarms in the alarm list. This provides users with more flexibility to customize the widget to their specific use case and makes it easier to focus on the relevant top-level alarms.
```

> The alarm list widget view is used to display alarms in dashboards and other places in the Cumulocity user interface. Previously, child alarms were always shown which could lead to information overload for users monitoring many alarms. With this change, a new toggle has been added to the widget configuration which allows showing or hiding the child alarms in the alarm list. This provides users with more flexibility to customize the widget to their specific use case and makes it easier to focus on the relevant top-level alarms.

Reviewer: ```suggestion
The "Alarm list" widget is used to display alarms in dashboards and other places in the {{< product-c8y-iot >}} UI. Previously, child alarms were always shown which could lead to information overload for users monitoring many alarms. With this change, a new toggle has been added to the widget configuration which allows showing or hiding the child alarms in the alarm list. This provides users with more flexibility to customize the widget to their specific use case and makes it easier to focus on the relevant top-level alarms.
```
